lib/logging.py

```diff
@@ -21,20 +28,30 @@ def logger() -> logging.Logger:
         return SecurityLog.seclogger
 
     @staticmethod
-    def logevent(request: HttpRequest, type: str, message: str) -> None:
-        # Unit tests do not have user defined in request object.
-        if request and hasattr(request, 'user'):
-            user=request.user
-            ip=request.META.get('REMOTE_ADDR')
-        else:
-            user='?'
-            ip='?'
-        SecurityLog.logger().info("({}/{}): {}: {}".format(
-            user,
-            ip,
-            type,
-            message
-        ))
+    def logevent(
+        request: HttpRequest,
+        type: str,
+        message: str,
+        severity: int=5,
+    ) -> None:
```

Comment:
```suggestion
    def logevent(
            request: HttpRequest,
            type: str,
            message: str,
            severity: int=5,
            ) -> None:
```
This style of indentation is suggested in the A+ Python style guide. The parameter list is indented one step further than the function body so that they stand out (when parameters are listed one per line).